src/tools/generate-copyright/src/licenses.rs

```diff
@@ -0,0 +1,48 @@
+//! A repository of supported licences
+
+/// Fetch a license text
+pub fn get(name: &str) -> Result<&'static str, anyhow::Error> {
+    if name == "0BSD" {
```

Comment:
The tool will always run inside of the repository and executed by bootstrap, so it can rely on the files being present on disk. I would just check if the file is present in the licenses directory and read it from there, rather than having to update the source code every time a new license is added.

Reply:
I thought about that, but I wasn't sure how I should calculate the path to the licences on disk, at run-time, given that I'm not sure what the CWD is when `x.py` is executed?